site/content/docs/developing/internal-albs.en.md

```diff
@@ -0,0 +1,89 @@
+
+## Environment
+
+The internal load balancer is an environment-level resource, to be shared among other permitted services. When you run `copilot env init`, you can import some specific resources to support the ALB. For services with `https` capability, use the [`--import-cert-arns`](../commands/env-init.en.md#what-are-the-flags) flag to import the ARNs of your existing private certificates. If you'd like your ALB to receive ingress traffic within the environment VPC, use the [`--internal-alb-allow-vpc-ingress`](../commands/env-init.en.md#what-are-the-flags) flag; otherwise, by default, access to the internal ALB will be limited to only Copilot-created services within the environment.
+
+!!!info
+    Today, Copilot will associate imported certs with an internal ALB *only if* the environment's VPC has no public subnets; otherwise, they will be associated with the default internet-facing load balancer. When initiating your environment, you may use the `--import-vpc-id` and `--import-private-subnets` flags to pass in your VPC and subnet IDs along with `--import-cert-arns`. For a more managed experience, you may use just the `--import-cert-arns` flag with `copilot env init`, then follow the prompts to import your existing VPC resources, opting out of importing public subnets. (Copilot's env config will soon have increased flexibility...stay tuned!)
+
+## Service
+
+The only service type that you can place behind an internal load balancer is a [Backend Service](https://aws.github.io/copilot-cli/docs/concepts/services/#backend-service). To tell Copilot to generate an internal ALB in the environment in which you deploy this service, add the `http` field to your Backend Service's workload manifest:
+
+```yaml
+# in copilot/{service name}/manifest.yml
+http:
+  path: '/'
+network:
+  vpc:
+    placement: private
+```
+
+!!!attention
+    Currently, you must use a new Backend Service that has not yet been deployed. Very soon, you will be able to add internal load balancers to existing Backend Services!
+
+## Advanced Configuration
+
+### Subnet Placement
+Maybe you would like your internal ALB and service(s) placed in different subnets than each other, or just want to specify subnet placement for one or the other or both.
+
+When you run `copilot env init`, use the [`--internal-alb-subnets`](../commands/env-init.en.md#what-are-the-flags) flag to pass in the IDs of the subnets in which you'd like the ALB to be placed.
+
+Call out specific subnets in your Backend Service manifest:
+
+```yaml
+# in copilot/{service name}/manifest.yml
+http:
+  path: '/'
+network:
+  vpc:
+    placement:
+      subnets: ["PrivateSubnetID1", "PrivateSubnetID2"]
+```
```

Comment:
I think this section might not be relevant for internal ALB (it's a more generic service feature). Can we remove it?